scripts/validate-common.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if grep -Rn Common/cpp -e '^#include "'; then
+    echo 'Found `#include "..."` in `Common/cpp/` directory. Convert it to `#include <...>`, e.g. `#include <reanimated/Fabric/updates/UpdatesRegistryManager.h>`.'
```

Comment:
Let's use something else as an example
```suggestion
    echo 'Found `#include "..."` in `Common/cpp/` directory. Convert it to `#include <...>`, e.g. `#include <reanimated/NativeModules/ReanimatedModuleProxy.h>`.'
```